src/cpp/hu.bme.mit.inf.modes3.ESP32-sensor/railsensor_3/ESP32-sensor/lib/readme.txt

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
Either keep this field up to date, or remove it from the repository. Should you decide on keeping the file, please consider reformatting it for Markdown style, since we use Github and it has built-in support for showing Markdown files online.